heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
+HOMEPAGE="http://hota.sourceforge.net/"
+COPYRIGHT="2004 Gil Megidish"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev = "8f1f525fcf50c4610302e870c85625c8ea2f33eb"
+SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$ srcGitRev.tar.gz"
+SOURCE_FILENAME = "heartofthealien- $ PORTVERSION ~ $ srcGitRev.tar.gz"
```

Comment:
No spaces before and after the `=` otherwise you will have errors.
This file is sourced like a shell script. If you add a space before the `=`, haikuporter will think you are launching a command.
And if you add a space after the `=` it thinks you have set the parameter to an empty string and then it thinks the rest of the line is a command.
So, the rule is: if you want spaces, they need to be inside a pair of quotes (or double quotes.)

And no spaces at all in PATCHES:
```
PATCHES="heartofthealien-$portVersion.patch"
```

Reply:
Edited.